services/rulesValidator.js

```diff
@@ -686,8 +686,13 @@ const validateSchema = (xmlString, schema, identifier, title, showDetails, lineO
             let errContext;
             const errorDetail = error;
             if ('line' in errorDetail) {
+                const lineMax = errorDetail.line >= 65535;
                 errorDetail.line += lineOffset;
-                errContext = `At line: ${errorDetail.line}`;
+                errContext = `At line${lineMax ? ' greater than' : ''}: ${errorDetail.line}${
+                    lineMax
+                        ? '. Note: The validator cannot display accurate line numbers for schema errors located at a line greater than 65535 in the activity.'
```

Comment:
Would it make sense to make this note variable, such that it says it cannot display accurate line numbers for lines greater than `${65535 + lineOffset}`? This would vary from file to file, but would be more accurate.

Reply:
Yea I think that's a good idea. I will update the language, etc.